pages/1.11/release-notes/1.11.9/index.md

```diff
@@ -0,0 +1,122 @@
+
+## GUI 
+- DCOS-40640, DCOS-45803 - In system tests, we use isolation to run tests in parallel. When we reuse `node_modules` it is important to link them in an isolation directory. In version 3.x, cypress caches its binary in a different folder. It is important to link the `.cache` folder. Cypress cache installation is required to speed up the build. Therefore, an automated bump of DC/OS UI to version v1.26.2 resolves this issue. 
+
+## Installing
+- DCOS-40442 - A fix to handle `AzureException` during storage upload.
+
+## Mesos 
+- COPS-4320, DCOS-46753 - This issue occurs due to a race between `.discard()` triggered by check container `TIMEOUT` and `IOSB` extracting `ContainerIO` object. This race could be exposed by overloaded/slow agent process if there are frequent check containers launched on an agent with heavy loads. This release fixes the issue by discarding `launch`, so the container I/O is cleaned up and therefore all FDs are closed.
+- DCOS-29474 - Occasionally flakes in `test_srv_records` are traced down to frameworks that are not correctly unregistered. This release validates that the framework is correctly unregistered, and throws an exception (triggering log collection) if the check fails after an uninstall. 
+- DCOS-46388 - If there is a validation error for `LAUNCH_GROUP`, or if there are multiple authorization errors for some of the tasks in a `LAUNCH_GROUP`, the master skips to process the remaining authorization results. Bump Mesos to nightly 1.7.x b97f0ba. 
+- DCOS-46574 - Few tests do not have a `maintainer` metadata. Example:`test_ldap_sync.py` does not show a `maintainer` tag. This release enforces up-to-date maintainer tag on all tests. 
+- DCOS-46814 - When an agent host reboots, all of its containers fail but the agent will try to recover from its checkpoint state after reboot. The agent will soon discover that all the `cgroup` hierarchies have failed and assume that the containers are destroyed. However, when trying to terminate the executor, the agent will first try to wait for the `exit` status of its container by `waitpid` on the checkpointed child process `pid`. If an agent host reboots and a new process with the same `pid` gets spawned then the parent waits for the wrong child process. This process will block the executor termination and future task status updates. Bump Mesos to nightly 1.5.x 5f6225b.
+- DCOS_OSS-4040 - The `dcos-diagnostic` bundle files are stored in `/var/lib/dcos/dcos-diagnostics/diag-bundles` and `/var/lib/dcos` is not a separate partition. It belongs to the root directory and there are many other critical data in the same directory on the masters, which could seriously damage a cluster if diagnostics bundle takes most of the space. The workaround is to move the destination directory (read volume) from root partition to a different partition than all these other directories which allows `dcos-diagnostics` bundles location to be configured. 
```

Comment:
I think a better description of this one would be "Add a configuration option that can be used to specify where to store diagnostic bundles on the masters"  Also, this is not a "Mesos" bugfix but rather an "Installing" bug fix as the change must be made at install time.